models.rdf

```diff
@@ -0,0 +1,93 @@
+      <rdfs:label xml:lang="en">Collection</rdfs:label>
+      <rdfs:comment xml:lang="en">
+        A Collection is a group of resources. Collections have descriptive metadata, access metadata,
+        and may links to works and/or collections. By default, member works and collections are an
+        unordered set, but can be ordered using the ORE Proxy class.
+      </rdfs:comment>
+      <rdfs:subClassOf rdf:resource="http://www.openarchives.org/ore/terms/Aggregation"/>
+      <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
+    </rdfs:Class>
+
+    <rdfs:Class rdf:about="http://pcdm.org/models#Object">
+      <rdfs:label xml:lang="en">Object</rdfs:label>
+      <rdfs:comment xml:lang="en">
+        An Object is an intellectual entity, sometimes called a "work", "digital object", etc.
+        Objects have descriptive metadata, access metadata, may contain files and other Objects as
+        member "components". Each level of a work is therefore represented by an Object instance,
+        and is capable of standing on its own, being linked to from Collections and other Objects.
+        Member Objects can be ordered using the ORE Proxy class.
+      </rdfs:comment>
+      <rdfs:subClassOf rdf:resource="http://www.openarchives.org/ore/terms/Aggregation"/>
+      <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
+    </rdfs:Class>
+   
+    <rdfs:Class rdf:about="http://pcdm.org/models#File">
+      <rdfs:label xml:lang="en">File</rdfs:label>
+      <rdfs:comment xml:lang="en">
+        A File is a sequence of binary data and is described by some accompanying metadata.
+        The metadata typically includes at least basic technical metadata (size, content type,
+        modification date, etc.), but can also include properties related to preservation,
+        digitization process, provenance, etc. Files MUST be contained by exactly one Object.
+      </rdfs:comment>
+      <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
+    </rdfs:Class>
+
+    <rdf:Property rdf:about="http://pcdm.org/models#hasFile">
+      <rdfs:label xml:lang="en">has file</rdfs:label>
+      <rdfs:comment xml:lang="en">Links to a File contained by this Object.</rdfs:comment>
+      <rdfs:domain rdf:resource="http://pcdm.org/models#Object"/>
+      <rdfs:range rdf:resource="http://pcdm.org/models#File"/>
+      <rdfs:subPropertyOf rdf:resource="http://www.w3.org/ns/ldp#contains"/>
+      <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
+    </rdf:Property>
+
+    <rdf:Property rdf:about="http://pcdm.org/models#hasMember">
+      <rdfs:label xml:lang="en">has member</rdfs:label>
+      <rdfs:comment xml:lang="en">Links to a related Object. Typically used to link to component parts, such as a book linking to a page.</rdfs:comment>
+      <rdfs:domain rdf:resource="http://www.openarchives.org/ore/terms/Aggregation"/>
+      <rdfs:range rdf:resource="http://www.openarchives.org/ore/terms/Aggregation"/>
```

Comment:
Ah yes, remembering now.

Continuing that thread here, I don't really like the idea of creating a class that we don't intend to be used directly. I know we do this with abstract classes and mixins, etc. in programming languages all the time, and that it's convenient, but I like the look of it a lot less in RDF since there's no way to hide or anonymize that class.

That said, I'd rather see the range/domain be something in the PCDM namespace, so maybe that's what it has to be. @escowles said:

> The problem with using fcdm:Resource as the range of fcdm:hasMember is it would allow Objects to have member Collections, which we don't want.

If we were to invent `pcdm:Resource`, would this be solved by making the range `pcdm:Object` but the domain `pcdm:Resource`?